build.sh

```diff
@@ -135,9 +135,10 @@ if (( ${NUMARGS} == 0 )) || hasArg libcuml || hasArg prims; then
 
     cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
           -DCMAKE_CXX11_ABI=${BUILD_ABI} \
-          -DBLAS_LIBRARIES=${INSTALL_PREFIX}/lib/libopenblas.a \
+          -DBLAS_LIBRARIES=${INSTALL_PREFIX}/lib/libopenblas.so.0 \
```

Comment:
Isn't this a pretty big change that potentially introduces a new runtime dependency?

Reply:
No, this is only a build time thing for building faiss, we don't use the host blas things